### ISSUE TITLE
Data Type params get truncated from CLI

### ISSUE THREAD

**Issue by GiaJordan** [Sage-Bionetworks/schematic#839]
**Describe the bug**
When validating a manifest from the CLI and specifying the dataType from the same, instead of the `config.yml`, the datatype gets truncated to the first letter and an error gets raised.

**To Reproduce**
1. Validate manifest and specify dataType from the CLI, for example `-dt MockComponent`
2. The below error is encountered

```
Exception has occurred: ValueError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
'M' is not a valid component in the schema.
```

**Expected behavior**
Full dataType string should be used without truncation if appropriate

**Comment by GiaJordan** [comment]
**cc:** @ychae

**Comment by mialy-defelice** [comment]
PR #831 should have fixed this error.
Though PR #836 also looks like a good solution. Just trying to make sure you have all the latest updates on your develop.

**Comment by GiaJordan** [comment]
@mialy-defelice I still get the error message on develop, but it should be up to date. It's showing the last merged PR as #815

**Comment by mialy-defelice** [comment]
@GiaJordan that's odd that your latest merged PR was #815. Can you try pulling it again? The other fixes in #831 are important and will need to be included in your develop branch.

**Comment by GiaJordan** [comment]
@mialy-defelice Sorry for the confusion! #831 was present on develop when I experienced it. At the time of the last comment, the previous three merges were 815, 817, and 831. 831 just wasn't the most recent
